lib/arena.rb

```diff
@@ -1,2 +1,47 @@
+# require "pry"
+
 class Arena
+  attr_accessor :name, :gladiators
+
+  def initialize (name)
+    @name = name[0].upcase + name[1..-1]
+    @gladiators = []
+  end
+
+  def add_gladiator (gladName)
+    @gladiators = @gladiators.push(gladName)
+    @gladiators = @gladiators.first(2)
```

Comment:
this works but what about checking for a condition, and then only pushing upon the successful check of that condition?